gschem/src/gschem_binding_integer.c

```diff
@@ -132,8 +132,8 @@ class_init (GschemBindingIntegerClass *klass)
 {
   GObjectClass *object_class = G_OBJECT_CLASS (klass);
 
-  klass->parent_class.update_model = update_model;
-  klass->parent_class.update_widget = update_widget;
```

Comment:
Something about casting specific to only two these functions. I cannot reproduce it any more (tried in several branches with various options I used before). Probably this has been fixed some other way. I'll revert the commit.